resources/views/layouts/app.blade.php

```diff
@@ -9,28 +9,24 @@
 
     <title>{{ config('app.name', 'Laravel') }}</title>
 
-        <!-- Fonts -->
+    <!-- Fonts -->
     <link rel="dns-prefetch" href="//fonts.bunny.net">
     <link href="https://fonts.bunny.net/css?family=Nunito" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" integrity="sha512-z3gLpd7yknf1YoNbCzqRKc4qyor8gaKU1qmn+CShxbuBusANI9QpRohGBreCFkKxLhei6S9CQXFEbbKuqLg0DA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 
     <!-- Font Aweosome-->
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" integrity="sha512-z3gLpd7yknf1YoNbCzqRKc4qyor8gaKU1qmn+CShxbuBusANI9QpRohGBreCFkKxLhei6S9CQXFEbbKuqLg0DA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 
-    <!-- Scripts -->
+    <!-- Style -->
     @vite(['resources/sass/app.scss', 'resources/js/app.js'])
     @yield('styles')
 
-    {{-- Styles --}}
+    <!-- Styles -->
     <link rel="stylesheet" href="{{asset('css/style.css')}}">
 
-    {{-- Font awesome --}}
+    <!-- Font awesome -->
```

Comment:
This one as well, we can remove this since we have the import already on line 15.